### PR TITLE
[DNM] Add new use of funds to banners

### PR DESCRIPTION
**Pull request by gbirke**
See https://phabricator.wikimedia.org/T266114

Changes will be merged with individual PRs into banners

### CODE REVIEW

**Comment by gbirke** [comment]
@Abban @moiikana I'll squash this PR and put it in the DE/EN/wp.de desktop banners when it's reviewed.

File: shared/components/ui/use_of_funds/FundsModal.pcss [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+			margin-left: $modal_padding;
+		}
+	}
+	&__section--two-cols {
+		@media ( min-width: $breakpoint_m ) {
+			display: flex;
+			margin: 0 -$modal_padding;
+		}
+	}
+	&__section--orgchart {
```
Comment (Abban) [inline shared/components/ui/use_of_funds/FundsModal.pcss:75]: I'd leave it flex on the lower size here, the image looks good next to the text with the smaller size

```scss
&__section--orgchart {
        display: flex;
        padding-top: $modal_padding;
	@media ( min-width: $breakpoint_m ) {
		font-size: 18px;
	}
}
```

File: shared/components/ui/use_of_funds/FundsModal.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+					</div>
+					<div className="banner_modal__section banner_modal__section--two-cols">
+						<div className="banner_modal__column">
+							<div className="banner_modal__column_info banner_modal__column_info--benefits_list">
+								Einen genauen Einblick in unsere internationalen Aktivitäten finden Sie hier:<br/>
+								<a href="#" target="_blank">Jahresplan Wikimedia Foundation</a>
+							</div>
+							<div className="banner_modal__benefits_list">
+								<h2>Was Ihre Spende bewirkt</h2>
+								<ul className="banner_modal__icon-list">
```
Comment (Abban) [inline shared/components/ui/use_of_funds/FundsModal.jsx:49]: The icons overlap a little on some sizes, but I guess there's not really much we can do about that.

File: shared/components/ui/use_of_funds/FundsModal.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+									entwickelt.
+								</p>
+
+							</div>
+						</div>
+						<div className="banner_modal__orgchart_image">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/2/2e/WMDE-funds-forwarding.gif"/>
+						</div>
+					</div>
+					<div className="banner_model__section banner_modal__section--call_to_action">
```
Comment (Abban) [inline shared/components/ui/use_of_funds/FundsModal.jsx:140]: Call to action on the modal 🎉

**Review by Abban — APPROVED** [review]
It seems like making this fully responsive would be a case of swapping the tabbed bar with an accordian on lower sizes so hopefully that ticket will be a little shorter.

**Comment by gbirke** [comment]
Closing this because it's an outdated starting point

**Pull request closed without merging**